### PR TITLE
[CPU] Enable Sum postop fusion for FullyConnected node

### PR DESCRIPTION
**Pull request by EgorDuplensky**
Very similar to Convolution Sum postop implementation

### Tickets
- 87721

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/fullyconnected.h [hunk 1]
```diff
@@ -63,12 +66,15 @@ class FullyConnected : public Node {
...
     VectorDims makeDummyOutputDims(const VectorDims& inDims) const;
 
+    InferenceEngine::Precision fusedEltwisePrecision(const Node& fusingNode) const;
+
     VectorDims inDims;
     VectorDims outDims;
 
     void setPostOps(dnnl::primitive_attr &attr, const VectorDims &dims, bool initWeights = false);
 
     bool withBiases = false;
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/fullyconnected.h:76]: @todo initialize with initializer list

File: src/plugins/intel_cpu/src/ngraph_transformations/snippets_mark_skipped.cpp [hunk 2]
```diff
@@ -248,6 +248,7 @@ bool isSuitableChildForFusingMatMul(const std::shared_ptr<const Node> &node, Nod
 bool isSuitableParentForFusingSumActivation(const std::shared_ptr<const Node> &node) {
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/ngraph_transformations/snippets_mark_skipped.cpp:248]: todo: rename

File: src/plugins/intel_cpu/src/nodes/eltwise.cpp [hunk 3]
```diff
@@ -2078,15 +2078,16 @@ bool Eltwise::canBeInPlace() const {
 
 void Eltwise::fuseInto(NodePtr& parentNode) {
     // Handling Convolution custom Add node fusing case which is processed via dnnl append_sum() API.
-    specialConvolutionAddFusing = (parentNode->getType() == Type::Convolution
-                                    || parentNode->getType() == Type::BinaryConvolution)
-                                        && getAlgorithm() == Algorithm::EltwiseAdd &&
-            dimsEqualWeak(getInputShapeAtPort(0).getDims(), getInputShapeAtPort(1).getDims());
+    specialConvolutionAddFusing = one_of(parentNode->getType(), Type::Convolution, Type::BinaryConvolution, Type::FullyConnected) &&
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/eltwise.cpp:2081]: todo: rename specialConvolutionAddFusing

**Comment by EgorDuplensky** [comment]
Todo: add tests

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Pull request closed without merging**